zerver/lib/actions.py

```diff
@@ -3049,8 +3049,9 @@ def do_send_confirmation_email(invitee, referrer, body):
     """
     activation_url = Confirmation.objects.get_link_for_object(invitee, host=referrer.realm.host)
     context = {'referrer': referrer, 'custom_body': body, 'activate_url': activation_url}
-    send_email('zerver/emails/invitation', invitee.email, from_email=settings.DEFAULT_FROM_EMAIL,
-               context=context)
+
+    from_email = "%s <%s>" % (referrer.full_name, settings.DEFAULT_FROM_EMAIL.split()[1])
```

Comment:
I think I'm also on the side of `%s (Zulip) <%s>`, or (via Zulip), or something.